returnn/oggzip.py

```diff
@@ -78,20 +81,26 @@ def run(self):
         if self.segments is not None:
             args.extend(["--subset_segment_file", tk.uncached_path(self.segments)])
 
-        if self.no_conversion:
-            args.extend(["--no_conversion"])
+        if self.no_audio:
+            args.extend(["--no_ogg"])
         else:
-            if self.rasr_cache is not None:
-                args.extend(["--sprint_cache", tk.uncached_path(self.rasr_cache)])
-            if self.raw_sample_rate is not None:
-                args.extend(["--raw_sample_rate", str(self.raw_sample_rate)])
-            if self.feat_sample_rate is not None:
-                args.extend(["--feat_sample_rate", str(self.feat_sample_rate)])
+            if self.no_conversion:
```

Comment:
Why the extra indentation? Seems to me you can put the check for self.no_conversion in an elif statement.